recipes/oracle.rb

```diff
@@ -58,6 +58,20 @@
   not_if { platform_family?('mac_os_x') }
 end
 
+username = nil
+password = nil
+
+if node['java']['oracle_user_databag_item'] &&
+   node['java']['oracle_user_databag'] &&
+   node['java']['oracle_user_databag_secret']
+
+  secret = Chef::EncryptedDataBagItem.load_secret(node['java']['oracle_user_databag_secret'])
```

Comment:
any secret management should be done outside of this cookbook and not reference a 3rd party company or integration you are using (datadog)